coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -203,7 +203,7 @@ def _extract_doc_comment_from_line(content, line, column, regex,
         if doc_comment:
             return end_line, end_column, doc
         else:
-            malformed_comment = MalformedComment(dedent("""\
+            malformed_comment = MalformedComment(dedent("""
```

Comment:
I think this will make whatever intended to do here became changed, maybe they dont want the newline character in this docs (not sure though)